<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PPSMonitorWidget</class>
 <widget class="QWidget" name="PPSMonitorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="cutter::StationButtonBar" name="station_btn_bar_" native="true"/>
   </item>
   <item>
    <widget class="QStackedWidget" name="monitor_stack_">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="none_monitor_"/>
     <widget class="cutter::HyperMonitorWidget" name="hypertherm_monitor_"/>
     <widget class="QWidget" name="koike_monitor_"/>
     <widget class="cutter::LiuheMonitorWidget" name="liuhe_monitor_"/>
     <widget class="cutter::MuskPPSMonitorWidget" name="musk_monitor_"/>
     <widget class="QWidget" name="kjellberg_monitor_"/>
     <widget class="QWidget" name="thermadyne_monitor_"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>cutter::MuskPPSMonitorWidget</class>
   <extends>QWidget</extends>
   <header>cutter/widget/IOMonitorWidget/PPSMonitor/MuskPPSMonitorWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>cutter::HyperMonitorWidget</class>
   <extends>QWidget</extends>
   <header>cutter/widget/IOMonitorWidget/PPSMonitor/hyper/HyperMonitorWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>cutter::LiuheMonitorWidget</class>
   <extends>QWidget</extends>
   <header>cutter/widget/IOMonitorWidget/PPSMonitor/liuhe/LiuheMonitorWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>cutter::StationButtonBar</class>
   <extends>QWidget</extends>
   <header>cutter/widget/StationButtonBar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
